.github/workflows/regression-tests.yml

```diff
@@ -200,6 +200,11 @@ jobs:
       continue-on-error: ${{matrix.may_fail || false}}
       timeout-minutes: 15
       run: |
+        export
+        if [ ${{matrix.self-hosted}} == "true" ]; then
+          module load "${{ matrix.sim }}"
```

Comment:
Usually `module` is an alias or bash function. Is the script where they are defined sourced? Alternatively, you can call `modulecmd` directly.